### PR TITLE
chore: move spec files to subfolders

### PR DESCRIPTION
**Pull request by chrisleewilcox**
## **Description**
See issue #[1213](https://github.com/MetaMask/mobile-planning/issues/1213)

## **Related issues**

Fixes: #

## **Manual testing steps**
Trigger any of the tests workflows in bitrise and they should continue to pass.

### **Before**

image

### **After**

image

## **Pre-merge author checklist**

- [x] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've clearly explained what problem this PR is solving and how it is solved.
- [x] I've linked related issues
- [x] I've included manual testing steps
- [x] I've included screenshots/recordings if applicable
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [x] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.
- [ ] I’ve properly set the pull request status:
  - [ ] In case it's not yet "ready for review", I've set it to "draft".
  - [ ] In case it's "ready for review", I've changed it from "draft" to "non-draft".

## **Pre-merge reviewer checklist**

- [x] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [x] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

File: e2e/specs/wallet/delete-wallet.spec.js [hunk 1]
```diff
@@ -1,22 +1,22 @@
 'use strict';
```
Comment (cortisiko) [inline e2e/specs/wallet/delete-wallet.spec.js:2]: I would say move this test to settings since it goes through the settings page.

**Review by cortisiko — DISMISSED** [review]
Looks good. Only left one minor comment

File: e2e/specs/wallet/delete-wallet.spec.js (hunk 1 above)
Comment (chrisleewilcox) [reply e2e/specs/wallet/delete-wallet.spec.js]: Done.

**Review by cortisiko — APPROVED** [review]
Looks good. Maybe as a follow-up task, we move all the failing/broken tests into a quarantine folder.

**Pull request merged**